src/Chapter3.hs

```diff
@@ -480,6 +511,45 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city totally.
 -}
 
+data Castle = MkCastle String deriving (Show)
+data Facility = Church | Library deriving (Show)
+data Inhabitants = One | Two | Three | Four deriving (Show)
+data House = MkHouse Inhabitants deriving (Show)
```

Comment:
Now, after you learned about newtypes, you can notice that `House` can be a newtype :slightly_smiling_face: 
```suggestion
newtype House = MkHouse Inhabitants deriving (Show)
```